tests/B2CWebAppCallsWebApi/Client/Services/TodoListService.cs

```diff
@@ -119,17 +119,19 @@ public async Task<IEnumerable<Todo>> GetAsync()
             throw new HttpRequestException($"Invalid status code in the HttpResponseMessage: {response.StatusCode}.");
         }
 
-        private async Task PrepareAuthenticatedClient()
+        private async Task PrepareAuthenticatedClient(string userFlow)
```

Comment:
```suggestion
        private async Task PrepareAuthenticatedClient(string? userFlow)
```

Reply:
this isn't enabled yet in the sample, will do later.